ui/shared/StatusProgress.jsx

```diff
@@ -6,20 +6,21 @@ import { getPercentComplete } from '../helpers/display';
 
 const StatusProgress = (props) => {
   const {
-    counts: { success, testfailed, busted, running, pending },
```

Comment:
Ah, because `testfailed` was changed a while back to include lint and build ("busted") failures in the API, so that is correcting for that: https://github.com/mozilla/treeherder/pull/7110/files#diff-a2ae2d06a8de37dbf79fc283a9983f35461aa5962fab9828e5833a939af2dce1R279

Will add a comment so its clearer though.

Reply:
Ah, I remember that now! Sounds good.